laterpay/asset_sources/scss/components/_debugger.scss

```diff
@@ -71,121 +77,105 @@
 
     .lp_debugger__close-link {
         color: $clickable;
-        font-weight: bold;
+        display: inline-block;
         height: $fs--2;
         line-height: 2;
-        margin: $fs--025 $fs--025 0 0;
         text-align: center;
-        transform: rotate(90deg);
+        transform: rotate(90deg) translate(2px, 3px);
         transition: transform 0.2s;
         width: $fs--2;
 
-        &:before {
-            line-height: 2;
-            padding: 0 0 0 ($fs * 0.1);
-            top: auto;
-        }
-
         &:hover {
             color: $clickable--highlight;
         }
-
-        .lp_debugger.lp_is-hidden & {
-            transform: rotate(-90deg);
-        }
     }
-}
 
 
-// debugger pane tabs --------------------------------------------------------------------------------------------------
-.lp_debugger-tabs {
-    border-bottom: 1px solid $border--darker;
-    margin: $fs--075 $fs--075 0;
-    position: relative;
-    z-index: $zindex;
-}
+    // debugger pane tabs --------------------------------------------------------------------------------------------------
+    .lp_debugger-tabs {
+        border-bottom: 1px solid $border--darker;
+        margin: $fs--075 $fs--075 0;
+        position: relative;
+        z-index: $zindex;
+    }
 
-.lp_debugger-tabs__link {
-    color: $clickable;
-    display: block;
-    padding: $fs--05 $fs;
+    .lp_debugger-tabs__link {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `lp_debugger-tabs__link` should be written in lowercase with hyphens